daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -1281,3 +1281,102 @@ def _resume_device(self):
         _da_set_logo_brightness(self, logo_brightness)
         _da_set_scroll_brightness(self, scroll_brightness)
         self.disable_notify = False
+
+
+class RazerMambaElite(__RazerDeviceSpecialBrightnessSuspend):
+    """
+    Class for the Razer Mamba Elite
+    """
+    USB_VID = 0x1532
+    USB_PID = 0x006C
+    HAS_MATRIX = True
+    WAVE_DIRS = (1, 2)
+    MATRIX_DIMS = [1, 20]
+    METHODS = [
+        'get_device_type_mouse',
+        'max_dpi',
+        'get_dpi_xy', 'set_dpi_xy',
+        'get_poll_rate', 'set_poll_rate',
+        # Logo logo_led_brightness/logo_matrix_effect_breath/...
+        'get_logo_brightness', 'set_logo_brightness',
+        'set_logo_wave',
+        'set_logo_static_mamba_elite',
+        'set_logo_spectrum_mamba_elite',
+        'set_logo_none_mamba_elite',
+        'set_logo_reactive_mamba_elite',
+        'set_logo_breath_random_mamba_elite', 'set_logo_breath_single_mamba_elite', 'set_logo_breath_dual_mamba_elite',
+        # Scroll wheel scroll_led_brightness/scroll_matrix_effect_breath/...
+        'get_scroll_brightness', 'set_scroll_brightness',
+        'set_scroll_wave',
+        'set_scroll_static_mamba_elite',
+        'set_scroll_spectrum_mamba_elite',
+        'set_scroll_none_mamba_elite',
+        'set_scroll_reactive_mamba_elite',
+        'set_scroll_breath_random_mamba_elite', 'set_scroll_breath_single_mamba_elite', 'set_scroll_breath_dual_mamba_elite',
+        # Left side left_led_brightness/left_matrix_effect_breath/...
+        'get_left_brightness', 'set_left_brightness',
+        'set_left_wave', 'set_left_static', 'set_left_spectrum', 'set_left_none', 'set_left_reactive',
+        'set_left_breath_random', 'set_left_breath_single', 'set_left_breath_dual',
+        # Right side right_led_brightness/right_matrix_effect_breath/...
+        'get_right_brightness', 'set_right_brightness',
+        'set_right_wave', 'set_right_static', 'set_right_spectrum', 'set_right_none', 'set_right_reactive',
+        'set_right_breath_random', 'set_right_breath_single', 'set_right_breath_dual',
+
+        # Can set all items (Logo,Scroll,Left and Right sides matrix_brightness/matrix_effect_custom/matrix_custom_frame
+        # emulate left/right effect
+        'set_brightness',
+        'set_custom_effect', 'set_key_row',
+        'get_brightness_mamba_elite',
+        'set_wave_effect_mamba_elite',
+        'set_static_effect_mamba_elite',
+        'set_spectrum_effect_mamba_elite',
+        'set_none_effect_mamba_elite',
+        'set_breath_random_effect_mamba_elite', 'set_breath_single_effect_mamba_elite', 'set_breath_dual_effect_mamba_elite',
+        'set_reactive_effect_mamba_elite']
+
+    DPI_MAX = 16000
+
+    DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1390/1390_mamba_elite.png"
+
+    # Deprecated - RAZER_URLS be removed in future.
+    RAZER_URLS = {
+        "top_img": DEVICE_IMAGE
```

Comment:
good idea, I wonder why I've never done that for other devices^^

Reply:
Wellcome 8-)